.github/pull_request_template.md

```diff
@@ -7,18 +7,53 @@ v    If your PR doesn't close an issue, that's OK!  Just remove the Closes: #XXX
 
 Closes: #XXX
 
-## Description
+## What is the purpose of the change
 
-<!-- Add a description of the changes that this PR introduces and the files that
-are the most critical to review.
--->
+*Add a description of the overall background and high level changes that this PR introduces *
 
-______
+*(E.g.: This pull request improves documation of area A by adding ....*
 
-For contributor use:
 
-- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
-- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
-- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
-- [ ] Re-reviewed `Files changed` in the Github PR explorer
+## Brief change log
 
+*(for example:)*
+ 
+  - *The metadata is stored in the blob store on job creation time as a persistent artifact*
+  - *Deployments RPC transmits only the blob storage reference*
+  - *Daemons retrieve the RPC data from the blob cache*
+
+
+## PR Formatting
+
+
+- Is PR targeted against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))  (yes / no)
+
+
+
+## Testing and Verifying
+
+*(Please pick one of the following options)*
+
+This change is a trivial rework / code cleanup without any test coverage.
+
+*(or)*
+
+This change is already covered by existing tests, such as *(please describe tests)*.
+
+*(or)*
+
+This change added tests and can be verified as follows:
+
+*(example:)*
+  - *Added unit test that validates ...*
+  - *Added integration tests for end-to-end deployment with ...*
+  - *Extended integration test for ...*
+  - *Manually verified the change by ...*
+
+## Documentation and Release Note
+
+  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
```

Comment:
Can this entire section be converted to a checklist? Similar to what we do for the SDK.